### PR TITLE
Drop python 2.7

### PR DESCRIPTION
**Pull request by northernSage**

### CODE REVIEW

**Comment by nicoddemus** [comment]
This is looking good to me, anything else you plan to add here (given this is a Draft)?

**Comment by northernSage** [comment]
I usually keep PRs as drafts until I have the time to look them over a last time to ensure I'm not missing anything. Feedback while I'm working is always nice as well.

**Review by nicoddemus — APPROVED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -13,7 +13,6 @@ classifiers=
     Intended Audience :: Developers
     License :: OSI Approved :: MIT License
     Operating System :: OS Independent
```
Comment (nicoddemus) [inline setup.cfg:15]: Dang just realized this now: we should set `python_requires` here.

Comment (northernSage) [reply setup.cfg:15]: Yep, totally missed it too. I'll submit a second PR later today to add it and will take the chance to also fix some minor typos and little things of the sort.

Comment (nicoddemus) [reply setup.cfg:15]: Cool, thanks!

Comment (northernSage) [reply setup.cfg:15]: Done (#41). On a side note, I wonder if we are ready for 0.15.0 or it would be better to wait a little more.

Comment (nicoddemus) [reply setup.cfg:15]: Thanks!

I'm OK with a making a new release, up to you. 👍

**Pull request merged**